Source/hu_stuff.c

```diff
@@ -566,8 +574,16 @@ void HU_Start(void)
   // [FG] fix crash when gamemap is not initialized
   if (gamestate == GS_LEVEL && gamemap > 0)
```

Comment:
This `gamemap > 0` check is redundant, we have moved it to `VANILLAMAP()`. This prevents E1M0 from showing at least a generic map name on the automap.